### ISSUE TITLE
load model issue

### ISSUE THREAD

**Issue by Su-study** [tascj/kaggle-hubmap-hacking-the-human-vasculature#2]
Congratulations! Thanks for your sharing!
when I run the code in kaggle notebook

`from mmengine.config import Config`
`from mmdet.apis import init_detector, inference_detector`
`cfg = Config.fromfile("/kaggle/input/rtdet-mask/config.py")`
`checkpoint_file = "/kaggle/input/rtdet-mask/iter_22400.pth" `
`model = init_detector(cfg, checkpoint=checkpoint_file, device="cuda:0") `
`result = inference_detector(model, imgs=path_list)  `

![截图 2023-08-06 21-23-58](https://github.com/tascj/kaggle-hubmap-hacking-the-human-vasculature/assets/50105534/c40b4dfd-6f2c-4d73-aef4-a4121ddf90fc)
![截图 2023-08-06 21-24-07](https://github.com/tascj/kaggle-hubmap-hacking-the-human-vasculature/assets/50105534/7a41accf-3f0a-455a-a377-a422491382ef)

The above config file is the same as you shared
I'm confused which file or config needs to be adjusted. Looking forward to your suggestion.

**Comment by tascj** [comment]
Probably `init_detector` does not work with the `MultiEMADetector` wrapper.

Here's how I prepare for submissions.

1. extract weights

```
import torch
import mmengine
from mmengine.runner import load_checkpoint
from mmdet.utils import register_all_modules
from mmdet.registry import MODELS

import custom_modules
register_all_modules()

exp_name = 'r0'
n_iter = 15120
ema_id = 2

cfg = mmengine.Config.fromfile(f'configs/{exp_name}.py')
model = MODELS.build(cfg.model)

load_checkpoint(model, f'work_dirs/{exp_name}/iter_{n_iter}.pth')
torch.save(dict(state_dict=model.ema_models[ema_id].state_dict()), f'work_dirs/{exp_name}/best.pth')
```

2. remove the `MultiEMADetector` wrapper in the config.
